test/api/add-po-box.test.js

```diff
@@ -17,7 +18,15 @@ test('tribes.addPOBox', t => {
       t.true(isPoBox(poBoxId), 'adding P.O. Box to group returns poBoxId')
 
       pull(
-        server.createUserStream({ id: server.id, reverse: true, limit: 1, private: true }),
+        server.db.query(
+          where(and(
+            author(server.id),
+            isDecrypted('box2')
```

Comment:
I think `private: true` in db1 means "please auto-decrypt" not "just private ones plz" like it does in db2.
If test passses I don't care :heart: